cmd/podman/system/reset.go

```diff
@@ -0,0 +1,82 @@
+package system
+
+import (
+	"bufio"
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/containers/libpod/cmd/podman/registry"
+	"github.com/containers/libpod/cmd/podman/validate"
+	"github.com/containers/libpod/pkg/domain/entities"
+	"github.com/containers/libpod/pkg/domain/infra"
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+)
+
+var (
+	systemResetDescription = `Reset podman storage back to default state"
+
+  All containers will be stopped and removed, and all images, volumes and container content will be removed.
```

Comment:
we say pods below, might want to add here.  Also, rather than "container content", would "container build cache" be better?